### ISSUE TITLE
Profit factor calculation

### ISSUE THREAD

**Issue by RatkoJ** [ranaroussi/quantstats#10]
Is there any reason why you calculate profit factor as
```python
return abs(returns.sum() / returns[returns < 0].sum())
```

as opposed to:
```python
return abs(returns[returns > 0].sum() / returns[returns < 0].sum())
```

I thought normally profit factor is `total profit / total loss`

**Comment by ranaroussi** [comment]
The formula is ``gross profit (total pnl) / gross loss``

**Comment by ranaroussi** [comment]
See this:
https://www.investopedia.com/articles/fundamental-analysis/10/strategy-performance-reports.asp

**Comment by RatkoJ** [comment]
Exactly. 
In there, the Gross Profit is $149,020. This corresponds to only the winning trades which can be checked by the `Avg. winning trade * num winners = 1460.98*102 = 149,020`. And in this case the profit factor is `Gross Profit / Gross Loss = 149,020/75,215 = 1.98`

**Comment by ranaroussi** [comment]
You are correct! Fixed in 0.0.23
